src/senaite/core/catalog/client_catalog.py

```diff
@@ -42,6 +42,7 @@
     "getClientID",
     "getName",
     "getEmailAddress",  # used in reference widget columns
+    "group_id",
```

Comment:
Regarding the camel case convention for columns vs. the snake case for indexes, shall we provide a getter for this property? Otherwise it might get confusing maybe.
But let me know what you think.

Reply:
Sure, going to add a `getGroupId` function

Reply:
> Regarding the camel case convention for columns vs. the snake case for indexes, shall we provide a getter for this property? Otherwise it might get confusing maybe. But let me know what you think.

Addressed with https://github.com/senaite/senaite.core/pull/2527/commits/2f6ea70bda4af4299f5d1d4c4918b699607811c0